docs/lib/predictions/fragments/android/identify-entity.md

```diff
@@ -123,10 +122,37 @@ public void detectEntities(Bitmap image) {
 }
 ```
 
+</amplify-block>
+<amplify-block name="Kotlin">
+
+```kotlin
+fun detectEntities(image: Bitmap) {
+    Amplify.Predictions.identify(
+        IdentifyActionType.DETECT_ENTITIES,
+        image,
+        Consumer { result: IdentifyResult ->
+            val identifyResult =
+                result as IdentifyEntityMatchesResult
+            val match = identifyResult.getEntityMatches()[0]
```

Comment:
Is there a `getBestMatch()` or something?

Reply:
Negative.